.github/workflows/integration.yaml

```diff
@@ -176,9 +176,9 @@ jobs:
           NEXTEST_RETRIES: 2
           CHAIN_COMMAND_PATHS: icad
         run: |
-          nix shell .#python .#ica -c \
```

Comment:
Please revert the modification. The upstream repo hermes using `nextest` to run tests. We should follow them.